presto-product-tests/bin/product-tests-suite-8-non-generic.sh

```diff
@@ -14,7 +14,7 @@ suite_exit_code=0
 
 TESTS_HIVE_VERSION_MAJOR="3" TESTS_HIVE_VERSION_MINOR="1" presto-product-tests-launcher/bin/run-launcher test run \
    --environment singlenode-hdp3 \
-    -- -g hdp3_only,storage_formats,hive_transactional \
+    -- -g storage_formats -x hdp3_only,hive_transactional \
```

Comment:
why do we need -x here?

Reply:
I saw that some tests are assigned to both `storage_formats` and `hive_transactional` test groups. With `-g storage_formats` in one job and `-g hdp3_only,hive_transactional` in another wouldn't they be executed in both?

Reply:
It seem there are no tests in both `storage_formats` and ` hdp3_only`. So we can drop `hdp3_only` at least from`-x` arg.
(Even if there was such we should use `profile_specific_tests` for filtering out, not `hdp3_only`)